cves/CVE-2019-14235.yml

```diff
@@ -49,7 +49,9 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  When passing certain inputs the django.utils.encoding.uri_to_iri conversion goes into excessive recursion re url encoding for 
```

Comment:
There seems to be a lot of project-specific jargon in this description. Could you explain what some of this is in more layman's terms?